cmd/server-main.go

```diff
@@ -65,6 +66,9 @@ ENVIRONMENT VARIABLES:
      MINIO_ACCESS_KEY: Custom username or access key of minimum 3 characters in length.
      MINIO_SECRET_KEY: Custom password or secret key of minimum 8 characters in length.
 
+  ENDPOINTS:
+     MINIO_ENDPOINTS: List of all the remote endpoints delimited by ' '.
```

Comment:
Since even local endpoints will work AFAICS, shouldn't this be `List of all the endpoints delimited by ' '.`